modules/oe_translation_poetry/tests/Functional/PoetryTranslationRequestTest.php

```diff
@@ -275,6 +287,302 @@ public function testUnprocessedJobHandling(): void {
     $this->assertJobsPoetryRequestIdValues($this->jobStorage->loadMultiple(), $expected_poetry_request_id);
   }
 
+  /**
+   * Tests the handling of Poetry status notifications.
+   */
+  public function testStatusNotifications(): void {
+    $this->prepareRequestedJobs([
+      'title' => 'My node title',
+      'field_oe_demo_translatable_body' => 'My node body',
+    ], ['fr', 'de', 'it']);
+
+    $identifier_info = [
```

Comment:
We could probably define a "defaultIdentifier" parameter in the setup, but not required.

Reply:
OK will do.